extras/systemd/foreman.socket

```diff
@@ -2,11 +2,7 @@
 Description=Foreman HTTP Server Accept Sockets
 
 [Socket]
-# Note Puma must be configured to listen on the same IP/port. If there's a
-# mismatch, it will bind again. The port can already be in use, for example when
-# systemd is configured with [::]:3000 but Puma with 0.0.0.0:3000. The port is
-# then already in use.
-ListenStream=0.0.0.0:3000
+ListenStream=3000
```

Comment:
See [man 5 systemd.socket](https://www.man7.org/linux/man-pages/man5/systemd.socket.5.html). By default just a port is interpreted as listen to it via IPv6 and then `BindIPv6Only` determines if it's dual stack (which it is by default).